doc/source/basics.rst

```diff
@@ -220,47 +240,38 @@ received bytes as Commands and to ensure that incoming and outgoing bytes abide
 by the protocol.
 
 .. ipython:: python
+    
+    class OurQueue:
```

Comment:
Standard queue would work just as well, though this may reinforce that it can be custom? I don't mind either way.